include/xtensor-r/rcontainer.hpp

```diff
@@ -21,9 +21,40 @@
 #include <Rcpp.h>
 #include <RcppCommon.h>
 
+// Fake type for R logicals
+struct rlogical {};
+
+// This handles the conversion of xtensor back to R type
+// When an xtensor with a T of rlogical is seen, the retrieved rtype
+// is LGLSXP (even though it is stored internally as an int in xtensor)
+namespace Rcpp
+{
+    namespace traits
+    {
+        template<> struct r_sexptype_traits<rlogical>{ enum{ rtype = LGLSXP }; };
```

Comment:
I know this code is adapted directly from Rcpp, but we should still stick to our formatting. 

It would also be interesting if the more modern "using" version of this would work:

```
template<> struct r_sexptype_traits<rlogical>
{
     static constexpr int rtype = LGLSXP;
}
```

Or otherwise, just adding some spaces should work fine:

```
template<> struct r_sexptype_traits<rlogical>
{
     enum
     {
           rtype = LGLSXP
     };
};
```

Reply:
Your modern suggestion seems to work fine. I'll keep that one.